Content.Shared/Damage/DamageGroupPrototype.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Collections.Generic;
+using Robust.Shared.IoC;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization;
+using Robust.Shared.Serialization.Manager.Attributes;
+
+namespace Content.Shared.Damage
+{
+    /// <summary>
+    /// A Group of DamageTypes.
+    /// </summary>
+    [Prototype("damageGroup")]
+    [Serializable, NetSerializable]
+    public class DamageGroupPrototype : IPrototype, ISerializationHooks
+    {
+        [Dependency] private IPrototypeManager _prototypeManager = default!;
+
+        [DataField("id", required: true)] public string ID { get; } = default!;
+
+        [DataField("damageTypes", required: true)]
+        public List<string> TypeIds { get; } = default!;
+
+        void ISerializationHooks.AfterDeserialization()
+        {
+            _prototypeManager = IoCManager.Resolve<IPrototypeManager>();
+        }
```

Comment:
This shouldn't be necessary. If it is something is tragically wrong. Either way, dependencies should not be getting resolved inside the AfterDeserialization hook.
```suggestion
```